docs/book/features/vsphere_csi_migration.md

```diff
@@ -0,0 +1,122 @@
+          }
+
+Please note that during a cluster upgrade we must drain each node (remove running workloads) before changing the configuration on the Kubelet.
+
+There is also an optional `CSIMigrationvSphereComplete` flag that can be enabled if all of the nodes have CSI migration enabled.
+`CSIMigrationvSphereComplete` helps stop registering the vSphere in-tree plugin in kubelet and volume controllers and enables shims and translation logic to route volume operations from the vSphere in-tree plugin to vSphere CSI plugin. Requires `CSIMigration` and `CSIMigrationvSphere` feature flags enabled and vSphere CSI plugin installed and configured on all nodes in the cluster.
+
+To verify the `CSIMigrationvSphere` feature flag is enabled and vSphere CSI driver installed on a particular node you can get the CSINode object. You should see `kubernetes.io/vsphere-volume` for an annotation `storage.alpha.kubernetes.io/migrated-plugins`.
+
+    # kubectl get csinodes -o yaml
+    apiVersion: v1
+    items:
+    - apiVersion: storage.k8s.io/v1
+      kind: CSINode
+      metadata:
+        annotations:
+          storage.alpha.kubernetes.io/migrated-plugins: kubernetes.io/vsphere-volume
+
+To verify vSphere in-tree PVCs and PVs are migrated to vSphere CSI driver, verify `pv.kubernetes.io/migrated-to: csi.vsphere.vmware.com` annotations is present on PVCs and PVs.
+
+PVC Annotations
+
+        Annotations:   pv.kubernetes.io/bind-completed: yes
+                       pv.kubernetes.io/bound-by-controller: yes
+                       pv.kubernetes.io/migrated-to: csi.vsphere.vmware.com
+                       volume.beta.kubernetes.io/storage-provisioner: kubernetes.io/vsphere-volume
+
+PV Annotations
+
+        Annotations:     kubernetes.io/createdby: vsphere-volume-dynamic-provisioner
+                         pv.kubernetes.io/bound-by-controller: yes
+                         pv.kubernetes.io/migrated-to: csi.vsphere.vmware.com
+                         pv.kubernetes.io/provisioned-by: kubernetes.io/vsphere-volume
+
+## Current limitations
+
+- vSphere CSI Driver does not support Provisioning `eagerzeroedthick` and `zeroedthick` volume. After the migration is enabled, when a new volume is requested using the in-tree provisioner and `diskformat` parameter, `diskformat` parameter is dropped by vSphere CSI Driver.
+- vSphere CSI Driver does not support raw vSAN policy parameters. After the migration is enabled, when a new volume is requested using in-tree provisioner and vSAN raw policy parameters specified in the StorageClass, these raw vSAN policy parameters are dropped by vSphere CSI Driver.
+- After in-tree vSphere volume is migrated to vSphere CSI driver, the volume can not be expanded even though CSI driver supports volume expansion.
+- vSphere CSI Migration requires vSphere 7.0u1. Customers who have in-tree vSphere volumes must upgrade vSphere to 7.0u1. Customers who do not need to migrate in-tree vSphere volumes can use vSphere 67u3 and above.
```

Comment:
Since this has been an issue in the past, can we not check in this PR until we have figured out the support matrix?